.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+name: Build and Publish
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build-publish:
+    name: Build and Publish
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+
+      - name: Set up Environment
+        run: |
+          python -m pip install --upgrade pip
+          python commands.py setup-env
+
+      - name: Install Dependencies to Local
+        run: python commands.py setup-env-to-lib
+
+      - name: Compile all languages
+        run: python commands.py compile
```

Comment:
Again this won't update anything, and with the right folder structure in place it shouldn't affect this Action.

Reply:
Translation addes, so I think it is used from now.